papers/ReTraCk/README.md

```diff
@@ -130,8 +130,9 @@ We conduct experiments over two datasets: [GrailQA](https://dl.orangedox.com/Wya
 4. Run the demo parser of interest (e.g., GrailQA)
    
    ```bash
-   cd parser 
-   python demo.py --config_path ./retrack/configs/demo_grailqa.json
+   # cd parser 
+   export PYTHONPATH=.
```

Comment:
If this is required, it should be part of the setup in the beggining of the README and not here.

Reply:
Yes, move it to the beggining.